tests/spec_decode/test_spec_decode_worker.py

```diff
@@ -101,7 +101,7 @@ def test_correctly_calls_target_model(k: int, batch_size: int):
         proposal_probs=proposal_probs,
         proposal_lens=proposal_lens)
 
-    exception_secret = 'artifical stop'
+    exception_secret = 'artificial stop'
```

Comment:
unrelated, but I found this anyway, so just fixing it here

Reply:
typo killer! 🎯